### PR TITLE
english/dream/lucid.md

### PR DESCRIPTION
**Pull request by SamuraiRanderson**
add lucid dream

### CODE REVIEW

**Comment by SamuraiRanderson** [comment]
Please review this, @casey-collab

File: english/dream/lucid.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+You’ve had some experience with lucid dreaming and begin to manipulate the dream so you have an escape from this
```
Comment (casey-collab) [inline english/dream/lucid.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/dream/lucid.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+marshmallow tomb. You intensely focus on the marshmallow walls around you. The wall in front of you begins to vibrate
+and slowly begins to separate. Through the opening in the wall you see a cobblestone path leading towards a castle
+in the distance.
+
+Do you:
+
+[Pinch yourself] (create-your-own-adventure/english/pinch/pinch.md)
```
Comment (casey-collab) [inline english/dream/lucid.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/lucid.md (hunk 1 above)
Comment (casey-collab) [inline english/dream/lucid.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: english/dream/lucid.md (hunk 1 above)
Comment (casey-collab) [inline english/dream/lucid.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/dream/lucid.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**